CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 3.6.0 (YYYY-MM-DD)
 
+### Breaking Changes
+
+* [ObjectServer] Read-only is not supported by synced Realm. "SyncConfigration.readOnly()" will always throw `UnsupportedOperationException`.
```

Comment:
Why do this? We have real use cases for it, it and it is easy to keep the checking in the Java layer for now?